bosh-director/lib/bosh/director/jobs/update_stemcell.rb

```diff
@@ -99,6 +105,13 @@ def perform
       
       private
 
+      def verify_sha1
+        stemcell_hash = Digest::SHA1.file(@stemcell_path).hexdigest
+        if stemcell_hash != @stemcell_sha1
+          raise StemcellSha1DoesNotMatch, "stemcell `#{@name}' SHA1 of `#{stemcell_hash}' does not match the expected value `#{@stemcell_sha1}'"
```

Comment:
change to:

Stemcell SHA1 '#{stemcell_hash}' does not match the expected SHA1 '#{@stemcell_sha1}